### PR TITLE
EnhancerAtlas enhancer adapter

### PR DESCRIPTION
**Pull request by dawit-melka**
- Implemented adapter for [EnhancerAtlas](http://enhanceratlas.org/downloadv2.php) dataset

### CODE REVIEW

File: config/schema_config.yaml [hunk 1]
```diff
@@ -73,8 +73,8 @@ epigenomic feature:
     source_url: str
 
 ontology term:
-  is_a: ontology class
   represented_as: node
+  is_a: ontology class
```
Comment (Habush) [inline config/schema_config.yaml]: why is this change here?

File: config/schema_config.yaml [hunk 2]
```diff
@@ -180,6 +180,9 @@ enhancer:
   inherit_properties: true
   properties:
     data_source: str
+  properties:
+    enhancer_type: str
+    snps: str[]
```
Comment (Habush) [inline config/schema_config.yaml]: what does the `snps` field describe?

File: config/adapters_config.yaml [hunk 3]
```diff
@@ -554,4 +554,32 @@ dbsuper_super_enhancer_regulates_gene:
...
+enhancer_atlas_enhancer_regulates:
+  adapter:
+    module: biocypher_metta.adapters.enhancer_atlas_adapter
+    cls: EnhancerAtlasAdapter
+    args:
+      enhancer_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer.bed.gz
+      enhancer_gene_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer_gene
+      tissue_to_ontology_filepath: ./aux_files/enhancer_atlas_tissues_to_ontology.pkl
+      type: enhancer to gene association
+      label: enhancer_gene
```
Comment (Habush) [inline config/adapters_config.yaml]: is there an `enhancer_gene` definition in the schema? Also shouldn't label be `regulates` ?

File: config/adapters_config.yaml [hunk 4]
```diff
@@ -554,4 +554,32 @@ dbsuper_super_enhancer_regulates_gene:
...
+  edges: True
+
+enhancer_atlas_enhancer:
+  adapter:
+    module: biocypher_metta.adapters.enhancer_atlas_adapter
+    cls: EnhancerAtlasAdapter
+    args:
+      enhancer_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer.bed.gz
+      enhancer_gene_filepath: /mnt/hdd_2/abdu/biocypher_data/enhancer_atlas/enhancer_gene
+      tissue_to_ontology_filepath: ./aux_files/enhancer_atlas_tissues_to_ontology.pkl
```
Comment (Habush) [inline config/adapters_config.yaml:577]: where is the `label` argument here?

**Review by Habush — CHANGES_REQUESTED** [review]

File: config/schema_config.yaml (hunk 1 above)
Comment (dawit-melka) [reply config/schema_config.yaml]: for most of the nodes the `represented_as` is defined first and have some consistency but I think it is better those changes be in the update prev adapters pr or other pr.

File: config/schema_config.yaml (hunk 2 above)
Comment (dawit-melka) [reply config/schema_config.yaml]: forgot to remove. thanks for pointing them out.

File: config/adapters_config.yaml (hunk 3 above)
Comment (dawit-melka) [reply config/adapters_config.yaml]: `enhancer_gene` is `input_label` and the output label is `regulates`

File: config/adapters_config.yaml (hunk 4 above)
Comment (dawit-melka) [reply config/adapters_config.yaml:577]: if label isn't present in the adapter config the default `enhancer` will be set in the adapter's constructor

File: config/adapters_config.yaml (hunk 3 above)
Comment (dawit-melka) [reply config/adapters_config.yaml]: I've changed `label` to `input_label`

File: config/schema_config.yaml [hunk 5]
```diff
@@ -181,6 +181,7 @@ enhancer:
   inherit_properties: true
   properties:
     data_source: str
+  properties:
```
Comment (Habush) [inline config/schema_config.yaml]: Duplicate properties field

File: biocypher_metta/adapters/enhancer_atlas_adapter.py [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+# Example enhancer atlas input file:
+
+# enhancer_file
+# chr1	875310	876520	11.4349429027718
+# chr1	876690	877510	10.6950350518485
+# chr1	902000	902080	10.429771917918
+# chr1	902180	902820	10.3902750575485
+
+# enhancer_gene_file
+# chr1:874840-876520_ENSG00000225880$LINC00115$chr1$762902$-	1.104330
```
Comment (Habush) [inline biocypher_metta/adapters/enhancer_atlas_adapter.py:22]: Can you add some text describing the last column? What do the numbers represent?

**Review by Habush — CHANGES_REQUESTED** [review]

**Review by Habush — APPROVED** [review]
Looks good!
